pallets/capacity/src/weights.rs

```diff
@@ -331,11 +345,24 @@ impl WeightInfo for () {
 		// Proof Size summary in bytes:
 		//  Measured:  `177`
 		//  Estimated: `6249`
-		// Minimum execution time: 43_000_000 picoseconds.
-		Weight::from_parts(45_000_000, 6249)
+		// Minimum execution time: 33_000_000 picoseconds.
+		Weight::from_parts(33_000_000, 6249)
 			.saturating_add(RocksDbWeight::get().reads(9_u64))
 			.saturating_add(RocksDbWeight::get().writes(6_u64))
 	}
+	/// Storage: `Capacity::ProviderBoostHistories` (r:1 w:1)
+	/// Proof: `Capacity::ProviderBoostHistories` (`max_values`: None, `max_size`: Some(641), added: 3116, mode: `MaxEncodedLen`)
+	/// Storage: `Capacity::ProviderBoostRewardPools` (r:7 w:0)
+	/// Proof: `Capacity::ProviderBoostRewardPools` (`max_values`: None, `max_size`: Some(113), added: 2588, mode: `MaxEncodedLen`)
+	fn claim_staking_rewards() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `1592`
+		//  Estimated: `20591`
+		// Minimum execution time: 90_000_000 picoseconds.
```

Comment:
This is worst case scenario.  Claiming rewards may be able to have some tuning, but also the benchmark should include variation.